features/openhab-addons-external/src/main/resources/conf/gcal.cfg

```diff
@@ -0,0 +1,10 @@
+# Credentials for Google API and Google Calendar name
+
+# Copied from Credentials page on console.developers.google.com
+#client_id=
+
+# Copied from Credentials page on console.developers.google.com
+#client_secret=
+
+# The name of the Google Calendar that GCal should use
+#calendar_name=
```

Comment:
Ah, I see now.  :-/  

I wasn't sure if that file was supposed to be updated.  I can do that.  Note that there also are some gcal-persistence parameters in openhab_default.cfg.  Can those be added to gcal.cfg (in a separate section of the file explaining their use for the persistence feature), or is a separate gcal-persistence.cfg needed?
```
#gcal-persistence:offset=
#gcal-persistence:executescript=
```
What about the Wiki?  It still references openhab.cfg and gcal: prefixes?

And, I see there are some log messages in the code that reference openhab.cfg...